src/ray/util/util_test.cc

```diff
@@ -0,0 +1,89 @@
+#include <signal.h>
```

Comment:
better rename this file to `signal_test.cc`. Because we may want to add tests for other components under util in the future.